src/Microsoft.Identity.Client/Platforms/Android/AndroidTokenCacheAccessor.cs

```diff
@@ -184,29 +184,5 @@ public MsalAccountCacheItem GetAccount(MsalAccountCacheKey accountKey)
         {
             return MsalAccountCacheItem.FromJsonString(_accountSharedPreference.GetString(accountKey.ToString(), null));
         }
-
-        /// <inheritdoc />
-        public int RefreshTokenCount => throw new NotImplementedException();
```

Comment:
I decided to remove these from the interface because they are actually only implemented on the InMemory stuff and are only used for test. It's dangerous to leave here, as one of use could use it, test on .net (which works), then realize that on Android and iOS it throws a NotImplEx....